# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dhcp/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dhcp-3.0.3/common/tr.c.vanilla	2005-10-21 15:27:01.000000000 +0200
+++ dhcp-3.0.3/common/tr.c	2005-10-21 15:38:38.950000000 +0200
@@ -55,6 +55,8 @@
  * As we keep a list of interesting routing information only, a singly
  * linked list is all we need
  */
+
+#define __u16 uint16_t
 struct routing_entry {
         struct routing_entry *next;
         unsigned char addr[TR_ALEN];
